fabric-chaincode-shim/src/main/java/org/hyperledger/fabric/ledger/Collection.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 IBM All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.fabric.ledger;
+
+/** Place holder. */
```

Comment:
Place holder?

Reply:
otherwise Checkstyle gets very very upset about an interface not having anything to do